db/migrate/20180410092453_policy_log.rb

```diff
@@ -0,0 +1,87 @@
+Sequel.migration do
+  tables = %i(roles role_memberships resources permissions annotations)
+
+  up do
+    execute """
+      CREATE TYPE policy_log_op AS ENUM ('INSERT', 'DELETE', 'UPDATE');
+      CREATE TYPE policy_log_kind AS ENUM #{literal tables.map(&:to_s)};
+      CREATE EXTENSION IF NOT EXISTS hstore;
+    """
+
+    key = %i(policy_id version)
+    create_table :policy_log do
+      String :policy_id, null: false
+      Integer :version, null: false
+      foreign_key key, :policy_versions, on_delete: :cascade
+      index key
+
+      column :operation, :policy_log_op, null: false
+      column :kind, :policy_log_kind, null: false
+      column :subject, :hstore, null: false
```

Comment:
Why is the data in an audit record called `subject`? That's really confusing to me.

Reply:
I'm confused as to what's confusing. It's the PK of the subject of this policy log record.